BBC.js

```diff
@@ -133,6 +141,20 @@ function scrape(doc, url) {
 				item.creators.push(ZU.cleanAuthor(authors[i], "author"));
 			}
 		}
+		else
+		{
+			var authorString = ZU.xpathText(doc, '//p[@class="byline"]');
+			var title = ZU.xpathText(doc, '//em[@class="title"]');
+			if (authorString) {
+				authorString = authorString.replace(title, '').replace('By', '');
+				var authors = authorString.split('&');
+				for (var i=0; i<authors.length; i++) {
+					item.creators.push(ZU.cleanAuthor(authors[i], "author"));
+				}
+			}	
+		}
```

Comment:
How about hard-coding the the blogname for newsbeat. 
```
if (url.indexOf("/newsbeat/article") != -1) {
  item.blogTitle = "BBC Newsbeat";
}
```

If that makes sense, please add&update test.